test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -40,6 +40,90 @@ using namespace iroha::ametsuchi;
 using namespace iroha::model;
 using namespace framework::test_subscriber;
 
+unsigned int SEED_ = 1337;
+const char LOWER_ALPHABET[] = "abcdefghijklmnopqrstuvwxyz";
+
+/**
+ * returns a number in a range [min, max)
+ */
+int64_t random_number(int64_t min, int64_t max) {
```

Comment:
Those functions not irrelevant for this file. Better to move them into helpers files.

Reply:
Ok. Also, this function is from `libs/transaction_utils/generator.h`. Some files in `libs` are not used anymore, so we have to remove them.

Reply:
Better to stay them as is. We remove all model-related stuff after migrating to shared_model.